### ISSUE TITLE
Sigma 19mm 2.8 EX DN ( E Mount ) + Sony A6000 ... combination not recognized

### ISSUE THREAD

**Issue by bronger** [lensfun/lensfun#607]
Hi.

I have a Sony A6000 and a Sigma 19mm 2.8 EX DN lens (the OG model).

According to the LensFun coverage page:

http://wilson.bronger.org/lensfun_coverage.html

... support is present for the lens.

Using darktable 1.6.2 and liblensfun 0.2.8, the camera/lens combo was not recognized at all from the RAW metadata.

I manually updated to liblensfun 0.3 and camera/lens auto detect would not work. I could manually select the camera/lens combo, though.

I noticed that the RAW EXIF data shows:

"E 19mm F2.8"

... as the lens model, but the liblensfun xml file shows:

"Sigma 19mm f/2.8 EX DN"

... as the lens model.

Relevant snippet:
    
    <lens>
        <maker>Sigma</maker>
        <model>Sigma 19mm f/2.8 EX DN</model>
        <mount>Sony E</mount>
        <mount>Micro 4/3 System</mount>
        <cropfactor>1.534</cropfactor>
        <calibration>
            
            <distortion model="poly3" focal="19" k1="-0.00748" />
            <tca model="poly3" focal="19" br="-0.0000466" vr="1.0003817" bb="0.0000346" vb="1.0000194" />
        </calibration>
    </lens>
    
Anyways, I opened a ticket on darktable redmine, lots more detail there:

http://www.darktable.org/redmine/issues/10331

They pretty much told me it's not a darktable bug, so I'm trying my luck here.

Basically, how can I force detection of the lens model?

Or is there nothing else to do on LensFun's side?

Thanks for your time.

Regards.

Reported by: rdeleon
